components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/APIGatewayManagerTest.java

```diff
@@ -92,6 +96,11 @@
             + "      \"url\":\"https://localhost:9443/am/sample/pizzashack/v1/api/\",\n" + "      \"config\":null\n"
             + "   }\n" + "}";
 
+    private String localEntry = "<localEntry key=\"" + apiUUId + "\">" +
```

Comment:
can't the variable value be written in 2 lines rather than 3?

Reply:
Can't do it since it exceeds the maximum